docs/HowToGuides/GettingStarted.md

```diff
@@ -138,7 +138,15 @@ toolchain as a one-off, there are a couple of differences:
    The required version of Xcode changes frequently and is often a beta release.
    Check this document or the host information on <https://ci.swift.org> for the
    current required version.
-2. Install [CMake][], [Ninja][] and [Sccache][]:
+1. **If you are running on Apple Silicon hardware** (M1, M2, etc):
```

Comment:
This looks good, but I wonder if the place for this is in the `### Spot check dependencies` section as a `**Note**`?

Reply:
Thanks for the quick reply!

I'm ok with either, I thought it made more sense here because it's a macOS-specific section, and because this is where my whole setup started going wrong :)

Turns out I had the x86 version of `brew` and it had installed the x86 versions of python, cmake, ninja & sccache too.

Reply:
Have moved it to the suggested location